census/models/utils.js

```diff
@@ -141,7 +141,11 @@ var processEntries = function(data, options) {
 
     data.pending = _.where(data.entries, {'isCurrent': false, 'reviewed': false});
     data.rejected = _.where(data.entries, {'isCurrent': false, 'reviewed': true, 'reviewResult': false});
-    _.remove(data.entries, function(e) { return e.isCurrent === false; });
```

Comment:
Ready
https://github.com/kravets-levko/opendatacensus/blob/feature/updates-to-api/tests/models.js#L53
https://github.com/kravets-levko/opendatacensus/blob/feature/updates-to-api/tests/models.js#L81